### DISCUSSION TITLE
135

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#67] [General]
# 潮流周刊第135期 - 香港不错

封面来源于上周在深圳参加 GIAC 以后，第二天去了一趟香港，下雨，随便逛了逛维多利亚港，随手拍了张照片，后面也逛了几个地方，挺不错，很潮流有文化的一个城市。

https://weekly.tw93.fun/posts/135-%E9%A6%99%E6%B8%AF%E4%B8%8D%E9%94%99

**Comment by LaceLetho** [comment]
话说你们去Hacker News看文章还是看评论？

**Reply by tw93** [reply]
瞎儿看
